test/lib/index.ts

```diff
@@ -60,3 +60,13 @@ test('.Net dotnet-invalid-manifest throws', async (t) => {
     includeDev),
   );
 });
+
+test('.Net core simple project tree generated as expected', async (t) => {
```

Comment:
Let's also add a test with:
- `includeDev = true`
- with missing deps property 
- invalid XML
 Can you think of some more as well?